src/property-filter/styles.scss

```diff
@@ -15,6 +15,8 @@ $operator-field-width: 120px;
 .search-field {
   display: flex;
   align-items: flex-end;
+  flex-wrap: wrap;
+  gap: awsui.$space-xs awsui.$space-s;
```

Comment:
1. Row spacing: For the vertical spacing when items are wrapped.
2. Column spacing: Replacing the `margin-inline-end` in line `179`